src/Form/DeveloperSyncForm.php

```diff
@@ -84,8 +84,18 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Synchronize developers'),
       '#open' => TRUE,
     ];
+
+    $output = '<p>' . $this->t('Developer synchronization will:') . '<p>';
```

Comment:
Build the content in a render array instead because that could be altered later if needed.
https://github.com/apigee/apigee-edge-drupal/blob/07c7fdccf84dab1de9d1e6c47e08e25de7595e4b/modules/apigee_edge_debug/src/Form/ConfigForm.php#L105